api/src/routers/reviewsRouter.js

```diff
@@ -0,0 +1,22 @@
+import express from "express";
+import { authenticate } from "../middlewares/authenticate.js";
+import {
+  getAllReviews,
+  getReviewById,
+  createReview,
+  updateReview,
+  deleteReview,
+} from "../controllers/reviewsController.js";
+
+const reviewsRouter = express.Router();
+
+//public routes
+reviewsRouter.get("/", getAllReviews);
+reviewsRouter.get("/:id", getReviewById);
+
+//protected routes
+reviewsRouter.post("/", authenticate, createReview);
```

Comment:
This can be a bit confusing, using the same endpoint to list ALL of the reviews and to create 1 single review. We usually separate them.